wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -122,6 +122,7 @@ public function get_instance_schema() {
 				'default' => '',
 				'sanitize_callback' => 'sanitize_text_field',
 				'description' => __( 'Title for the widget' ),
+				'should_preview_update' => false,
```

Comment:
Correct this is PR is very much like having a `shouldComponentUpdate`, or a connected component that only listens to certain changes in the state tree and re-renders accordingly.

Between here and there though, I still think we need this logic to prevent re-rendering the preview on each `keyup` in the title.  Especially if we are hitting API endpoints or oembed providers.